### PR TITLE
Add declarative color picker

### PR DESCRIPTION
**Pull request by KRLango**
This is to add a declarative version of the color picker for use in the line plot display layers section of the inspector

### CODE REVIEW

**Review by Tiomat85 — APPROVED** [review]

**Comment by KRLango** [comment]
@Ion-e this PR is ready to test.

**Review by Ion-e — APPROVED** [review]
Draft looks good, thanks!

File: nion/swift/Inspector.py [hunk 1]
```diff
@@ -4994,3 +5015,22 @@ def data_item_delete() -> None:
...
 
         return None
+
+
+class DeclarativeColorChooserConstructor:
+    def __init__(self, app: Application.Application) -> None:
+        self.__app = app
+
+    def construct(self, d_type: str, ui: UserInterface.UserInterface, window: typing.Optional[Window.Window], d: Declarative.UIDescription, handler: Declarative.HandlerLike, finishes: typing.List[typing.Callable[[], None]]) -> typing.Optional[UserInterface.Widget]:
+        if d_type == "color_chooser":
```
Comment (cmeyer) [inline nion/swift/Inspector.py:5025]: Like the other PR, can you make this a qualified name like "nionswift.color_chooser" to make it easier to migrate this code to `nionui` in the future? Thanks!

**Review by cmeyer — CHANGES_REQUESTED** [review]
Like the other PR, can you make this a qualified name like "nionswift.color_chooser" to make it easier to migrate this code to `nionui` in the future? Thanks!

**Review by Tiomat85 — APPROVED** [review]

**Comment by cmeyer** [comment]
Merged 4d1937b3c9c221f13c656215334e70c095daf163

**Pull request closed without merging**